app/models/question.rb

```diff
@@ -2,10 +2,33 @@ class Question < ApplicationRecord
 
   belongs_to :user
   has_many :answers, dependent: :destroy
+  has_many :topictaggings
+  has_many :topics, through: :topictaggings, dependent: :destroy
+  has_many :lecturetaggings
+  has_many :lectures, through: :lecturetaggings, dependent: :destroy
   has_one :correct_answer, ->{ where(correct: true) }, class_name: "Answer",
           foreign_key: "question_id"
 
   accepts_nested_attributes_for :answers, :correct_answer
   validates :correct_answer, presence: true
 
+	def all_topics=(names)
```

Comment:
i was following the tutorial. i don't really know. it was talking about how doing that an atribute of aa_topic will be added to the form data.